### ISSUE TITLE
Basemap Selector

### ISSUE THREAD

**Issue by jmoe** [trailheadlabs/carlessca#20]
Need to implement the base map selector. Which of the OuterSpatial basemaps do we want to use? I propose Hike, Topo and Satellite.

**Comment by ryanbranciforte** [comment]
yeah, those three work well. i suggest we use google hybrid for satellite version.  i think we need to have more options including google maps and np map option. 

gmaps b/c we put a lot of effort into getting transit data into gtfs 
np map team b/c we want to show partnership with nps 

sorry i missed adding this in the design!

**Comment by jmoe** [comment]
Let's launch it like this, then we can iterate.

It seems like we can create a basemap selector that is like the overlay selector, instead of having the basemaps be links under the map. That looks a little weird to me anyways.

But let's get it functional and get through everything else first.

I have deployed the first working version of the links.

**Comment by ryanbranciforte** [comment]
agreed, i like idea of having a basemap selector

**Comment by ryanbranciforte** [comment]
i like the NPS base maps in there. i think if we added a google maps option, we'd be good. (gmaps is nice b/c we promoted gtfs for this project and the transit stops show up on the map)
